lib/puppet/application/cert.rb

```diff
@@ -71,7 +71,12 @@ def subcommand=(name)
   end
 
   option("--[no-]allow-dns-alt-names") do |value|
-    options[:allow_dns_alt_names] = value
+    options[:allow_subject_alt_names] = value
```

Comment:
Need to be sure to update the places where this is pulled out of the options hash to use the new name.

Reply:
Looks like this one got updated to "subject" by accident.

Reply:
I thought we wanted this one to be subject so that they pointed to the same hash?